nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/explorer/grid-list/registry/nf-registry-grid-list-viewer.html

```diff
@@ -115,7 +115,39 @@
                                                 No comments specified
                                             </div>
                                             <div fxLayout="row" class="mat-caption">
-                                                {{(snapshotMeta.timestamp/1000 | amFromUnix) | amDateFormat:'MMM-DD-YYYY'}} at {{(snapshotMeta.timestamp/1000 | amFromUnix) | amDateFormat:'h:mm A'}}
+                                                {{(snapshotMeta.timestamp/1000 | amFromUnix) | amDateFormat:'YYYY-MM-DD'}} at {{(snapshotMeta.timestamp/1000 | amFromUnix) | amDateFormat:'h:mm A'}}
+                                            </div>
+                                        </div>
+                                    </td-step>
+                                </td-steps>
+                            </div>
+                            <div *ngIf="droplet.bundleType" id="nifi-registry-explorer-grid-list-viewer-droplet-container-details-change-log">
+                                <td-steps mode="vertical">
+                                    <td-step label="Version {{snapshotMeta.version}} ({{nfRegistryService.getHumanReadableSize(snapshotMeta.contentSize)}}) - {{snapshotMeta.timestamp | amTimeAgo}}" sublabel="by {{snapshotMeta.author}}"
+                                             *ngFor="let snapshotMeta of droplet.snapshotMetadata; let i = index"
+                                             [active]="i === 0 ? true : false">
+                                        <div fxLayout="column" fxLayoutAlign="space-between stretch">
+                                            <div fxLayout="row" class="mat-body-3">
+                                                <div>Build Information:</div>
+                                                <div style="margin-left: 14px; margin-bottom: 5px;">
+                                                    <b>NiFi API Version</b>: {{snapshotMeta.systemApiVersion}}<br />
```

Comment:
Instead of using `<br/>`, I believe it would be better to use one `<div>` per row.